README.md

```diff
@@ -59,7 +59,8 @@ Environment variables descriptions:
     * `recordings` - provides access to call recordings
     * `callHistory` - provides access to call history
     * `extensions` - retrieves the list of other users (contacts)
-    * `messaging` - provides access to sending and receiving messages
+    * `instantMessaging` - provides access to sending and receiving messages
```

Comment:
Let's rename this parameter from `instantMessaging` to `internalMessaging` to indicate this functionality related to internal instant messaging.